web/libs/ui/src/lib/code-block/code-block.tsx

```diff
@@ -1,15 +1,25 @@
 export function CodeBlock({
   code,
   className,
+  variant = "default",
 }: {
   title?: string;
   description?: string;
   code: string;
   className?: string;
+  variant?: "default" | "warning" | "negative";
 }) {
+  const variantStyles = {
+    default: "bg-neutral-surface border-neutral-border",
+    warning: "bg-warning-background border-warning-border-subtle",
+    negative: "bg-negative-background border-negative-border-subtle",
+  };
+
   return (
-    <div className={`whitespace-pre-wrap font-mono mt-2 p-3 bg-neutral-surface rounded-md ${className || ""}`}>
-      {code.trim()}
+    <div
+      className={`p-3 rounded-small border scrollbar-thin scrollbar-thumb-neutral-border-bold scrollbar-track-transparent ${variantStyles[variant]} ${className || ""}`}
```

Comment:
Can we use `cnm` helper here instead of the string concat, as that does not have any ability to properly order, merge or deal with tailwind classes, where the other helper does.

An example is here: https://github.com/HumanSignal/label-studio/blob/5836becb7ad47674f94d16b085308b0f0482125c/web/libs/editor/src/components/TaskSummary/SummaryBadge.tsx#L1